app/app/views/employees/show.html.erb

```diff
@@ -1,13 +1,14 @@
 <h2> Employee Details </h2> <br>
 <%= link_to 'Back', store_path(params[:store_id]) %>  
 <%= link_to 'Edit', edit_store_employee_path(params[:store_id], params[:id]) %>
+
 <section> 
 <div> Name: <%= @employee.name %> <div/>
 <div> Email: <%= @employee.email %> <div/>
 <div> Private number: <%= @employee.private_number %> <div/>
 <div> Position: <%= @employee.position %> <div/>
 <div> Id number: <%= @employee.id_employee %> <div/>
```

Comment:
All this `div` label are all wrong closing, please fix it like this way -> `</div>`

Reply:
resolve 👍